browse/services/database/__init__.py

```diff
@@ -164,3 +168,51 @@ def get_document_count() -> Optional[int]:
     document_count: int = db.session.query(Document).\
         filter(not_(Document.paper_id.like('test%'))).count()
     return document_count
+
+
+@db_handle_error(logger=logger, default_return_val=None)
+def get_sequential_id(paper_id: Identifier,
+                      context: str = 'all',
+                      is_next: bool = True) -> Optional[str]:
+    """Get the next or previous paper ID in sequence."""
+    local_session = db.session()
+    baked_query = bakery(lambda session: session.query(Document.paper_id))
```

Comment:
My use of the sqlalchemy bakery extension may be of interest to reviewers. I found this a useful mechanism for building database queries programmatically.

Reply:
Ya, that looks useful.